p2p/testutils/fn.go

```diff
@@ -0,0 +1,36 @@
+package testutils
```

Comment:
When you cross this comment, let's either jump into MEWS, or schedule a call in protocol hour. I feel strongly about deleting this file
<img width="815" alt="Screen Shot 2022-03-16 at 5 31 01 PM" src="https://user-images.githubusercontent.com/1892194/158713375-7513d294-4337-441f-a438-0117c70db076.png">
 because `mockgen` allows this to be configured at test configuration time.